SPECS/ruby/ruby.spec

```diff
@@ -66,8 +163,25 @@ rm -rf %{buildroot}/*
 %{_mandir}/man1/*
 %{_mandir}/man5/*
 %{_rpmconfigdir}/macros.d/macros.ruby
+%{_rpmconfigdir}/macros.d/macros.rubygems
+%{_rpmconfigdir}/fileattrs/rubygems.attr
+%{_rpmconfigdir}/rubygems.req
+%{_rpmconfigdir}/rubygems.prov
+%{_rpmconfigdir}/rubygems.con
+%dir %{rubygems_dir}
+%{rubygems_dir}/rubygems
 
 %changelog
+* Thu Mar 11 2021 Henry Li <lihl@microsoft.com> - 2.6.6-4
+- Upgrade to version 2.7.2
+- Add files like macros.rubygems, imported from Fedora 32 (license: MIT)
```

Comment:
Nit: here, can we list all of the files that we import from Fedora? Just so there's no ambiguity in the future regarding licenses for specific files, especially since Fedora was not the original source of the spec.